wherehows-web/app/components/dataset-compliance.js

```diff
@@ -7,14 +7,21 @@ const {
   get,
   isBlank,
   setProperties,
-  getWithDefault
+  getWithDefault,
+  String: { htmlSafe }
 } = Ember;
 
 // TODO: DSS-6671 Extract to constants module
 const missingTypes = 'Looks like some fields may contain privacy data ' +
   'but do not have a specified `Field Format`?';
 const successUpdating = 'Your changes have been successfully saved!';
 const failedUpdating = 'Oops! We are having trouble updating this dataset at the moment.';
+const hiddenTrackingFieldsMsg = htmlSafe(
```

Comment:
Agree for the thoughts on partial..We have some partial template files with one 1 line tho :)
But seems like easy to maintain.